src/components/Navigation/Pages/PageStatus/PageStatus.js

```diff
@@ -46,15 +46,17 @@ export default class PageStatus extends Component {
       this.fetchDefinitionAvailability(),
       this.fetchCrawledPerDay(),
       this.fetchRecentlyCrawled(),
-      this.fetchCrawlbreakdown()
+      this.fetchCrawlbreakdown(),
+      this.fetchToolsranperday()
```

Comment:
Let's change it to camel case here and everywhere `this.fetchToolsRanPerDay()`.

Reply:
@storrisi Does it make sense? Is there a particular reason for this casing?

Reply:
Sorry for the delay. I just copied the value coming from the backend, your change request definitely makes sense for consistency.